parlai/mturk/tasks/dealnodeal/run.py

```diff
@@ -0,0 +1,85 @@
+# Copyright (c) 2017-present, Facebook, Inc.
+# All rights reserved.
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree. An additional grant
+# of patent rights can be found in the PATENTS file in the same directory.
+import os
+import time
+from parlai.core.params import ParlaiParser
+from parlai.mturk.core.mturk_manager import MTurkManager
+from parlai.mturk.tasks.dealnodeal.worlds import \
+    MTurkDealNoDealDialogWorld
+from parlai.agents.local_human.local_human import LocalHumanAgent
+from task_config import task_config
+import copy
+from itertools import product
+from joblib import Parallel, delayed
+import threading
+
+"""
+This task consists of one agent talking to an MTurk worker
+"""
+def main():
+    argparser = ParlaiParser(False, False)
+    argparser.add_parlai_data_path()
+    argparser.add_mturk_args()
+    opt = argparser.parse_args()
+    opt['task'] = 'dealnodeal'
+    opt['datatype'] = 'valid'
+    opt.update(task_config)
+
+    mturk_agent_1_id = 'mturk_agent_1'
+    mturk_agent_2_id = 'mturk_agent_2'
+    human_agent_1_id = 'human_1'
+    mturk_agent_ids = [mturk_agent_1_id] #, mturk_agent_2_id]
```

Comment:
The hardcoded/commented option of using human agents vs using two mturk agents isn't as good form as it could be. Any chance this can be done programmatically and an additional flag can be added to the parser?

Can use `parser.add_argument('--use_human', dest='use_human', type=bool, default=False, action='store_true')` and then select the ids based on whether or not `opt['use_human']` is set.

Reply:
@mikelewis0 assuming this is just for other people to evaluate their models, you could do what I commented at the top-level.

Reply:
In which case I'd remove the unused second turk agent and comment. If you want them to be able to collect more data if they'd want to, you can do both as well (as they don't conflict)